ejb3/src/main/java/org/jboss/as/ejb3/component/singleton/SingletonComponentDescription.java

```diff
@@ -125,7 +125,9 @@ public void configure(final DeploymentPhaseContext context, final ComponentDescr
                     if(interceptorConfig.getPostConstruct() != null) {
```

Comment:
@dmlloyd The fact that the interceptorConfig does not cover `@Interceptor`
is another issue. Due to this `LifecycleCMTTxInterceptor.Factory` is called with null `MethodIdentifier` which leads to the default (NOT_SUPPORTED) transaction scope on SFSB regardless of the actual transaction attribute defined either on the bean or on the interceptor. In other words, if you set `REQUIRES_NEW` on a `@Interceptor` of SFSB it will be ignored and interceptor's lifecycle methods will always be executed within `NOT_SUPPORTED`. This violates the spec.
Note that I'd expect it to be covered by TCK.